includes/Checker/Checks/Plugin_Readme_Check.php

```diff
@@ -59,43 +59,57 @@ protected function check_files( Check_Result $result, array $files ) {
 			return;
 		}
 
-		// Check the readme.txt for default text.
+		// Check the readme file for default text.
 		$this->check_default_text( $result, $readme );
 
-		// Check the readme.txt for a valid license.
+		// Check the readme file for a valid license.
 		$this->check_license( $result, $readme );
 
-		// Check the readme.txt for a valid version.
+		// Check the readme file for a valid version.
 		$this->check_stable_tag( $result, $readme );
 	}
 
 	/**
-	 * Checks the readme.txt for default text.
+	 * Checks the readme file for default text.
 	 *
 	 * @since n.e.x.t
 	 *
 	 * @param Check_Result $result The Check Result to amend.
 	 * @param array        $files  Array of plugin files.
 	 */
 	private function check_default_text( Check_Result $result, array $files ) {
-		if (
-			self::file_str_contains( $files, 'Here is a short description of the plugin.' ) ||
-			self::file_str_contains( $files, 'Tags: tag1' ) ||
-			self::file_str_contains( $files, 'Donate link: http://example.com/' )
-		) {
+		$default_text_patterns = array(
+			'Here is a short description of the plugin.',
+			'Tags: tag1',
+			'Donate link: http://example.com/',
+		);
+
+		$file = '';
+		foreach ( $default_text_patterns as $pattern ) {
+			$file = self::file_str_contains( $files, $pattern );
+			if ( $file ) {
+				break;
+			}
+		}
+
+		if ( $file ) {
 			$result->add_message(
 				false,
-				__( 'The readme.txt appears to contain default text.', 'plugin-check' ),
+				sprintf(
+					/* translators: %s: readme file */
+					__( 'The %s appears to contain default text.', 'plugin-check' ),
+					str_replace( $result->plugin()->path( '/' ), '', $file )
+				),
 				array(
 					'code' => 'default_readme_text',
-					'file' => $result->plugin()->path( '/readme.txt' ),
+					'file' => $file,
```

Comment:
This needs to also be the path relative to the plugin's directory, similar to other checks.
```suggestion
					'file' => str_replace( $result->plugin()->path(), '', $file ),
```